pkg/plugins/models.go

```diff
@@ -103,6 +104,10 @@ func (e Includes) DashboardURLPath() string {
 	return "/d/" + e.UID
 }
 
+func (e Includes) IsRBACReady() bool {
+	return e.Action != ""
```

Comment:
This is a bit of a weird thing, but - if the preferred semantics here is that absence of the `action` field is equivalent to the empty string, then it's probably preferable to have the schema declare:

```cue
action: string | *""
```

this likely to be an ongoing and larger question around Go codegen and optionality; see https://github.com/grafana/grok/issues/1 for a bit of discussion

Reply:
I see what you mean, *absent* vs *empty*... :+1: 
I wonder :thinking: If plugin developers take the time to specify the action field to *empty*, what does it mean to them: *don't use RBAC* or *anyone can access*.
Let's see with an example, a user specifying:
```
  "includes": [
    {
      "type": "page",
      "name": "Tenants",
      "path": "/a/test-app?path=tenants",
      "role": "Editor",
      "action": "",
      "addToNav": true,
      "defaultNav": true
    },
```
Here which option do we understand:
1. when RBAC is disabled, only Editors can access this include but when RBAC is enabled, anyone can access it.
2. this include is not protected with RBAC, only Editors can access it.

Given option 2 is more secure in case of developer error, I'm going to go with option 2. But then I agree, I should rename the `IsRBACReady` function to `requireRBACAction`.

Reply:
i don't understand all the details here, but it sounds like a "fail closed" approach. [Which is good](https://community.microfocus.com/cyberres/b/sws-22/posts/security-fundamentals-part-1-fail-open-vs-fail-closed) :)